chart/values.yaml

```diff
@@ -73,9 +73,6 @@ config:
   # config.KINM_DB_CONNECTIONS -- Advanced - the number of connections in the database pool for kinm
   KINM_DB_CONNECTIONS: "5"
 
-  # config.OBOT_SERVER_IGNORE_INACTIVE -- whether the controller processes objects that have been marked as inactive
-  OBOT_SERVER_IGNORE_INACTIVE: false
-
```

Comment:
@drpebcak Will removing this have adverse effects on the terraform?

Reply:
should be fine, but we can remove it from there as well